--- origsrc/libdbi-drivers-0.8.3-1/drivers/mysql/Makefile.am	2011-11-10 14:16:38.985942700 -0600
+++ src/libdbi-drivers-0.8.3-1/drivers/mysql/Makefile.am	2011-11-10 14:15:55.118666200 -0600
@@ -6,9 +6,9 @@ if BUILDDOCS
 # don't use docdir, doc_DATA. This way we keep the HTML in separate subdirs
 # for each driver
 docdir = $(datadir)/doc/$(PACKAGE)-$(VERSION)
-htmldir = $(docdir)/dbd_mysql
+dbdhtmldir = $(docdir)/dbd_mysql
 doc_DATA = dbd_mysql.pdf
-html_DATA = dbd_mysql/* 
+dbdhtml_DATA = dbd_mysql/* 
 else
 docdir =
 htmldir =
--- origsrc/libdbi-drivers-0.8.3-1/drivers/pgsql/Makefile.am	2011-11-10 14:16:39.001542700 -0600
+++ src/libdbi-drivers-0.8.3-1/drivers/pgsql/Makefile.am	2011-11-10 14:15:24.901413500 -0600
@@ -4,9 +4,9 @@ if HAVE_PGSQL
 
 if BUILDDOCS
 docdir = $(datadir)/doc/$(PACKAGE)-$(VERSION)
-htmldir = $(docdir)/dbd_pgsql
+dbdhtmldir = $(docdir)/dbd_pgsql
 doc_DATA = dbd_pgsql.pdf
-html_DATA = dbd_pgsql/* 
+dbdhtml_DATA = dbd_pgsql/* 
 else
 docdir =
 htmldir =
--- origsrc/libdbi-drivers-0.8.3-1/drivers/sqlite3/Makefile.am	2011-11-10 14:16:39.001542700 -0600
+++ src/libdbi-drivers-0.8.3-1/drivers/sqlite3/Makefile.am	2011-11-10 14:15:38.083436500 -0600
@@ -6,9 +6,9 @@ if HAVE_SQLITE3
 
 if BUILDDOCS
 docdir = $(datadir)/doc/$(PACKAGE)-$(VERSION)
-htmldir = $(docdir)/dbd_sqlite3
+dbdhtmldir = $(docdir)/dbd_sqlite3
 doc_DATA = dbd_sqlite3.pdf
-html_DATA = dbd_sqlite3/* 
+dbdhtml_DATA = dbd_sqlite3/* 
 else
 docdir =
 htmldir =
